lib/src/main/java/com/frogggias/smarttable/commons/CSVHelper.java

```diff
@@ -69,7 +69,10 @@ public Uri createCSVFromCursor(String name, Cursor cursor, String[] columnNames,
                     if (i > 0) {
                         outputStream.write(mSeparator);
                     }
-                    outputStream.write(cursor.getString(cursor.getColumnIndex(columnNames[i])).getBytes());
+                    String stringFromCursor = cursor.getString(cursor.getColumnIndex(columnNames[i]));
+                    if (stringFromCursor != null) {
+                        outputStream.write(stringFromCursor.getBytes());
+                    }
```

Comment:
Maybe add `else` branch and write something like `"null"`?

Reply:
Maybe.